nservicebus/hosting/azure-functions/service-bus_configuration_ASBFunctions_[1.3,).partial.md

```diff
@@ -0,0 +1,5 @@
+NServiceBus can be registered and configured on the host builder using the `UseNServiceBus` extension method in the startup class:
+
+snippet: asb-function-default
+
+The default builder method will look up values such as function name, connection string and license information directly from from the `IConfiguration`. Fine-grain configuration is available through the overload accepting a `ServiceBusTriggeredEndpointConfiguration` class.
```

Comment:
The overload doesn't accept a `ServiceBusTriggeredEndpointConfiguration`. It accepts a factory method to create one.

We may be able to simplify things by just saying fine-grained config is also possible and then putting the asb-function-hostbuilder snippet in.

Reply:
Rephrased. See if it is better now?